CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 1.0.0
+Nov 10, 2023
+- Uses the stable v1 LMNT REST API.
+- (BREAKING) Default audio encoding format is now mp3 (previously wav). Please pin to a previous version if needed.
```

Comment:
Suggest putting all of these under a parent item as 'Breaking changes: please update your code to use the new behavior, or pin to a previous version if preferred." Then you can remove everywhere you put (BREAKING) prefix and "Please pin..." below.